solutions/core/template.yaml

```diff
@@ -174,20 +189,6 @@ Resources:
         - Key: Name
           Value: !Sub "${AWS::StackName}-DynamoDbSSEKey"
 
-  DynamoDbSSEKeyAlias:
```

Comment:
Reason for using aliases is it is much easy to identify keys in KMS otherwise all you have is the key id without any context - could we add all key aliases back please